shell/packages/accounts-email-token/token_server.js

```diff
@@ -53,8 +57,9 @@ Accounts.registerLoginHandler("emailToken", function (options) {
     };
   }
 
-  if (!user.services || !user.services.emailToken ||
-      !(user.services.emailToken.tokens)) {
+  var identity = user.identities[0];
```

Comment:
Does this mean I can't log in with my email token identity, if it's not the first identity for my user?

Also, consider using `identity.` instead of `user.identities[0]` below if you're going to pull it out into a variable.